db/seeds.rb

```diff
@@ -1,7 +1,33 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+require 'faker'
+
+puts "Cleaning database..."
+User.destroy_all
+Property.destroy_all
+
+puts "🌱 Seeding data..."
+puts "👩🏽‍🦱 Creating users with faker..."
+
```

Comment:
Besides these 10 random users, consider creating an user for each one of us (meleu, anne and ariel).

Just so we can easily log into the app and navigate through it.

Also create at least one property for each one of us.